h2o-k8s/README.md

```diff
@@ -0,0 +1,103 @@
+# H2O Kubernetes integration
+
+The integration of Kubernetes and H2O is possible via `water.k8s.KubernetesEmbeddedConfigProvider` - to be found
+in this module. This implementation of `EmbeddedConfigProvider` is dynamically loaded on H2O start and remains inactive
+unless H2O running in a Docker container managed by Kubernetes is detected.  
+
+## Running H2O in K8s - user's guide
+
+H2O Pods deployed on  Kubernetes cluster require a 
+[headless service](https://kubernetes.io/docs/concepts/services-networking/service/#headless-services) 
+for H2O Node discovery. The headless service, instead of load-balancing incoming requests to the underlying
+H2O pods, returns a set of adresses of all the underlying pods. It is therefore the responsibility of the K8S
+cluster administrator to set-up the service correctly to cover H2O nodes only.
+
+### Creating the headless service
+First, a headless service must be created on Kubernetes.  
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: h2o-service
+spec:
+  type: ClusterIP
+  clusterIP: None
+  selector:
+    app: h2o-k8s
+  ports:
+  - protocol: TCP
+    port: 54321
```

Comment:
What about internal H2O port? I'm sure it works, just curious :)

Reply:
Ah, I see, this this the port exposed to the outside word

Reply:
This is the port exposed to other K8S entities. The final exposure to the outside world is done by another layer named Ingress or Route (on OpenShift).